docs/deployment/docker.md

```diff
@@ -65,7 +65,13 @@ WORKDIR /app
 COPY yarn.lock package.json packages/backend/dist/skeleton.tar.gz ./
 RUN tar xzf skeleton.tar.gz && rm skeleton.tar.gz
 
-RUN yarn install --frozen-lockfile --production --network-timeout 300000 && rm -rf "$(yarn cache dir)"
+# install sqlite3 dependencies
+RUN apt-get update && \
+    apt-get install -y libsqlite3-dev python3 cmake g++ && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN yarn config set python /usr/bin/python3 && \
```

Comment:
What do you think about moving this up to the RUN above, so that it becomes a single command that it easy comment out/remove? I'm thinking that a common optimization that one might want to do is to move sqlite to be a `devDependency`, so that these it can be removed from the image. With that we could also clearly comment and tip that it's possible to remove the block and make it a dev dep.

That's assuming some conversion I was on Discord is correct, which mentioned that the extra dependencies added here add significant size to the final image.

Reply:
yeah, the size is the main problem. If you didn't include python before in your image, it could be +300 MB.
Ok, I'll move `yarn config set python /usr/bin/python3`to the previous run.